tests/test_distributions.py

```diff
@@ -200,11 +200,19 @@ def test_eq_ne_hash():
     d0 = distributions.UniformDistribution(low=1, high=2)
     d1 = distributions.UniformDistribution(low=1, high=3)
     assert d0 != d1
+    assert not d0 == d1
```

Comment:
How about adding `assert not d != copy.deepcopy(d)` right after `assert d == copy.deepcopy(d)` in [L196](https://github.com/optuna/optuna/pull/726/files#diff-f700676dbe171de91407c9cf58db8008R196) similarly to this test case?

Reply:
That'd be more consistent, fixed!